### ISSUE TITLE
Multi-class question

### ISSUE THREAD

**Issue by ZoraizQ** [facebookresearch/InvariantRiskMinimization#10]
Does the mean_accuracy and cross-entropy loss work with multi-class / categorical logits MLP out of the box?
